reconcile/utils/amtool.py

```diff
@@ -45,6 +46,19 @@ def config_routes_test(yaml_config: str, labels: Mapping[str, str]) -> AmtoolRes
     return result
 
 
+def version() -> AmtoolResult:
+    """Returns the version parsed from amtool --version"""
+    result = _run_cmd(["amtool", "--version"])
+
+    pattern = re.compile("^amtool, version (?P<version>[^ ]+) .+")
+    if m := pattern.match(result.message):
+        return AmtoolResult(True, m.group("version"))
+    else:
```

Comment:
this `else` branch isn't needed

Reply:
ah, yes, return false by default. Sure that's more idiomatic, fixed in d31e81f4